mlflow/sagemaker/__init__.py

```diff
@@ -354,14 +511,148 @@ def _deploy(role, image_url, app_name, model_s3_path, run_id, region_name):
         Tags=[
             {
                 'Key': 'app_name',
-                'Value': app_name,
+                'Value': endpoint_name,
             },
         ],
     )
-    eprint("endpoint_config_arn: %s" % endpoint_config_response["EndpointConfigArn"])
+    
```

Comment:
nit: want the newline below the `eprint`?

Reply:
Moved it after the `eprint` to create two logical blocks: one for endpoint configuration creation and logging, another for endpoint creation and logging